netbox/dcim/choices.py

```diff
@@ -1444,6 +1473,18 @@ class CableTypeChoices(ChoiceSet):
                 (TYPE_AOC, 'Active Optical Cabling (AOC)'),
             ),
         ),
+        (
+            _('USB'), (
+                (TYPE_USB_1, 'USB 1.0'),
+                (TYPE_USB_1_1, 'USB 1.1'),
+                (TYPE_USB_2, 'USB 2.0'),
+                (TYPE_USB_3, 'USB 3.0'),
+                (TYPE_USB_3_1, 'USB 3.1'),
+                (TYPE_USB_3_2, 'USB 3.2'),
+                (TYPE_USB_4, 'USB 4'),
+                (TYPE_USB_4_2, 'USB 4 2.0'),
```

Comment:
Not all of these specify a new cable type. I spent a few minutes looking over the [Wikipedia page](https://en.wikipedia.org/wiki/USB) but it wasn't clear to me what the different classes of USB cabling actually are. Let's just add a single generic "USB" type for now. It doesn't need to be in a separate subgroup; it can be listed before "power.")

Reply:
@jeremystretch I'm ook going to generic USB overall, but I would like to make a case for distinction between at least USB 2 and USB 3 series as the bandwidth capacity is vastly different with USB2 capping out at 480gb and USB3 series ranging between 5Gb and 20gb.  I would think it would be more in line with ok there various data capacity cables, but if the distinction isn't deemed important, I'll go ahead and make the suggested generic modifications.